web3/geth.py

```diff
@@ -74,14 +81,21 @@ class GethAdmin(ModuleV2):
     """
     https://github.com/ethereum/go-ethereum/wiki/Management-APIs#admin
     """
+    add_peer = add_peer
```

Comment:
Instead of interleaving these, maybe it would be good to keep all the non-deprecated ones together and to group all the deprecated ones below